docs/getting-started/integrating-codacy-with-visual-studio-code.md

```diff
@@ -1,90 +1,91 @@
+    The number next to each file name is the total number of Quality issues that the pull request adds to or removes from that file. The number farther to the right is the number of new Medium or Critical Quality issues. If there are any such issues, the file name is also highlighted in yellow (Medium) or red (Critical).
 
-        -   The number farther to the right of each file name is the number of issues with severity Medium or higher in that file. The file is also highlighted in yellow if there are any such issues.
+    ![Important issues](./images/codacy-vscode-extension-important-issues.png)
 
 -   The **diff coverage and coverage variation** introduced by the pull request. These are the same values you find in the [Pull request coverage overview](../repositories-coverage/pull-requests.md#coverage-overview) panel in the Codacy app.
 
-    !!! tip
-        The percentage next to each file name is the coverage variation for that file.
+    The percentage next to each file name is the coverage variation for that file.
 
 -   Sequences of **duplicate code** (clones). These are the same ones you find in the [Quality Duplication tabs](../repositories/pull-requests.md#duplication-tabs) in the Codacy app.
 
 -   Variations in **code complexity**.
 
-## Navigating to found problems
+## Reviewing pull request issues
+
+In the Problems tab, Codacy displays the same Quality issues you find in the main view and lets you navigate to the exact line of code where the issue was found.
 
-To navigate to a found problem:
+!!! note
+    Code coverage, duplicates, and complexity aren't currently shown in the Problems tab.
 
-1.  Open the **Problems tab** (use `Ctrl+Shift+M` on Windows/Linux or `Cmd+Shift+M` on macOS)
+To review Quality issues:
 
-1.  Click the name of the issue you wish to review.
+1.  Open the **Problems tab** (use `Ctrl+Shift+M` on Windows/Linux or `Cmd+Shift+M` on macOS).
 
-    Line-based navigation is currently only supported for Quality issues.
+1.  Click the name of the issue you want to review.
 
     ![Navigate to a specific issue from the Problems tab](images/codacy-vscode-extension-problems-tab.png)
 
-1.  Hover over a highlighted issue in the code editor to view more details and available quick fixes.
+1.  Hover over a highlighted issue in the code editor to view available actions and suggested quick fixes (if available).
 
     For a list of tools that support quick fixes, see [Supported languages and tools](./supported-languages-and-tools.md#supported-languages-and-tools)
 
-## Refreshing pull request data
-
-The Codacy extension refreshes the pull request data every time you push new changes to the Git provider, after the analysis is complete. The analysis usually takes less than 5 minutes.
+1.  Once you've addressed the problems in your code, push your changes to the Git provider so that Codacy analyzes the updated code. Once the analysis is complete, the Codacy extension automatically refreshes the pull request data.
```

Comment:
Suggestion to avoid having two "once". I think the second part can be put out of the procedure, as something that it's triggered after you execute the last step (push changes).
WDYT?

```suggestion
1.  Once you've addressed the problems in your code, push your changes to the Git provider so that Codacy analyzes the updated code.

When the analysis is complete, the Codacy extension automatically refreshes the pull request analysis result.
```